kong/templates/kong_defaults.lua

```diff
@@ -189,4 +189,5 @@ tracing_sampling_rate = 0.01
 wasm = off
 wasm_filters_path = NONE
 wasm_dynamic_module = NONE
+wasm_runtime = wasmtime
```

Comment:
I'm curious about this. `wasm_runtime` is decided at build time, so ideally-speaking we don't want a user-facing config param for it

Is it required for some reason? I see that it is used in the nginx template for deciding if we can inject directives for a particular wasm runtime. Does ngx_wasm_module throw an error if there's a mismatch? i.e. there's a `wasmer` directive in nginx.conf, but the wasm runtime is `wasmtime`?

Reply:
@flrgh Good thinking. I have quickly attempted to use the FFI to detect the runtime, but failed, so left it as an improvement. Do you know if there is an API for that, even if still not exposed through the FFI? We could use such an API to detect the runtime and ensure the Kong config is consistent with it.

Reply:
There's no non-hacky way that I can think of to retrieve this info at runtime currently, _but_ I don't think we need to anyways. I did some testing with this, and my finding is that having extraneous runtime blocks/directives in nginx.conf doesn't trigger any problems--empty or not. So, for instance, the following config does not produce any error:

```
wasm {

  module tests ./spec/fixtures/proxy_wasm_filters/build/tests.wasm;
  module response_transformer ./spec/fixtures/proxy_wasm_filters/build/response_transformer.wasm;

  wasmtime {
    flag debug_info on;
  }

  v8 {
  }

  wasmer {
    flag threads on;
  }

}
```

I think the `wasm_runtime` conf item can just be removed for now.

Reply:
> I think the wasm_runtime conf item can just be removed for now

I second this. Less code, less risks.

Reply:
Done!